### ISSUE TITLE
Protein activity checkbox in curation status does not 'check' WBPaper00053873

### ISSUE THREAD

**Issue by draciti** [WormBase/afp_webapp#22]
when the box protein activity in the Phenotypes and function widget is checked it does not show a ‘checked’ value in the afp value in the curation status form. @kimberly, can you confirm that this should be the expected behavior? i.e. having a ‘checked’ value in the catalyticact field in the curation status from when the ‘protein activity’ box is checked in AFP

**Comment by valearna** [comment]
the 'protein avtivity' checkbox is linked to 'invitro' and not 'catalyticact'. I used the tables defined here: https://docs.google.com/spreadsheets/d/1sS_uAjBJ2r5H90Lam62Ai0HunjwvfjnklkFNrDoNXeU/edit#gid=1601232973

Let me know if we need to use catalyticact instead of invitro.

**Comment by vanaukenk** [comment]
@valearna 

@draciti and I just took a look at this and we think this part of the form needs to be a bit clearer to make sure we exactly get the information we're looking for.

We would then also need to change the table where the data will go.

In the Phenotypes and function section, then, we would like to change the text:
Protein Activity (i.e., Enzymatic Activity) -> Enzymatic Activity

and have the resulting checkbox write to an afp_catalyticact table.  

I updated the [Google spreadsheet](https://docs.google.com/spreadsheets/d/1sS_uAjBJ2r5H90Lam62Ai0HunjwvfjnklkFNrDoNXeU/edit#gid=1601232973), but am not sure if an afp_catalyticact table exists, so we may need to create that.

Please let us know if anything is not clear.  Thx.

**Comment by valearna** [comment]
Ok, thanks @vanaukenk and @draciti. @azurebrd can you create the table afp_catalyticact both on mangolassi and tazendra?

**Comment by azurebrd** [comment]
@valearna is just talking to me, and I think I understand but want to make sure we're all on the same page.

There's SVM flagging for catalyticact but it's not in afp/cfp, so an exception was made to add it to the curation status form.  Is this right ?  Is there any other stuff for it besides SVM in the curation status form ?

There's afp/cfp but no SVM for invitro, and there's data there from 2008.  Is this the same data as should be in catalyticact ?  If so, do you want to get rid of invitro from afp/cfp and transfer the data into a new afp/cfp table catalyticact, then remove the exception from curation status ?

We're still kind of maintaining the CFP form, but is anyone using it or planning to use it ?  I think it'd be nice to get rid of it to avoid overhead of maintaining something not being used.

**Comment by vanaukenk** [comment]
Thanks @azurebrd 

Here's my understanding of the history of these flags:

The invitro flag has been around for a long time but was notoriously vague and wound up capturing a lot of different types of data, some of which were relevant for GO Molecular Function (MF) curation, others, not so much.  More recently, I've tried to come up with flagging methods for specific types of GO MFs, one of which was catalytic activities.  

The catalytic activity SVM was implemented after the current AFP form (and long after the existing cfp form), and we never changed either of those forms to reflect this 'new' data type.

However, for the new AFP form, I think it makes sense to be clearer and have the authors specifically flag catalytic activities and then write to a corresponding afp catalyticact tables (when it gets created).

Because of the lack of specificity in the invitro flagging, I don't think we want to merge the invitro and catalyticact tables, although I would keep the invitro tables and their data as a record of what kinds of things people flagged.  It may still help us hone in on other data types we should explore for future flagging.

As for the CFP form, we could ask curators whether they think we need to keep it.  As long as the data is still accessible either via querying the tables or the curation status form, I'm not sure we need the form.

Let me know if you need any more info.  Thx.

**Comment by azurebrd** [comment]
Thanks @vanaukenk !

It sounds like invitro and catalyticact then have data that is different and should be kept permanently separate. So we should create a new set of catalyticact tables, remove the exception for the curation status form, and then we'll be set ?

**Comment by vanaukenk** [comment]
@azurebrd 

Yes, I think we should create an afp_catalyticact table (and whatever other tables are needed to be compliant with the curation status form); and we'll continue to keep the data from invitro separate from catalyticact.

Thx.

**Comment by azurebrd** [comment]
Okay, thanks.  Created on tazendra, edited the curation status form to remove the exception.

**Comment by valearna** [comment]
Thanks @azurebrd !
I modified the label of the checkbox in the form - Protein Activity (i.e., Enzymatic Activity) -> Enzymatic Activity and I linked the checkbox to the new afp_catalyticact table. I tested the behavior of the checkbox for paper 00053873 - I marked it as 'Enzymatic Activity' positive and now I see it in the 'AFP positive' set in the curation status form for 'catalyticact'. I'm leaving the issue open for further testing by @draciti and @vanaukenk

**Comment by azurebrd** [comment]
cool, thanks @valearna !
